nx.json

```diff
@@ -0,0 +1,46 @@
+{
+	"$schema": "./node_modules/nx/schemas/nx-schema.json",
+	"installation": {
+		"version": "19.5.0",
+		"plugins": {
+			"@nx/js": "19.5.0",
+			"nx/plugins/package-json": "19.5.0"
+		}
+	},
+	"release": {
+		"groups": {
+			"images": {
+				"projects": [ "build/*" ]
+			},
+			"deploy": {
+				"projects": [ "deploy" ]
+			}
+		},
+		"projectsRelationship": "independent",
+		"version": {
+			"conventionalCommits": true
+		},
+		"changelog": {
+			"projectChangelogs": {
+				"createRelease": "github"
```

Comment:
Is there a reason to do Github releases? I'd vote for not adding them as I see no real benefit in having them for now...

Reply:
I generally think those release pages will be a helpful asset to making release announcements, but don't feel strongly about it. I would prefer to keep this turned-on for now so we can review how they feel on our first releases this sprint. We can always remove the entry/entries and turn it off if we don't like it.

Reply:
Agree. This is not the decision to maintain github releases from now on. Let's test it and evaluate afterwards. 👍🏾